t/043-shdict.t

```diff
@@ -2398,53 +2398,23 @@ type: table
 
 
 
-=== TEST 91: dogs, cat mixing
+=== TEST 91: duplicate zones
 --- http_config
     lua_shared_dict dogs 1m;
-    lua_shared_dict cats 1m;
---- config
-    location = /test {
-        content_by_lua '
-            local dogs = ngx.shared.dogs
-            dogs:set("foo", 32)
-            dogs:set("bah", 10502)
-            local val = dogs:get("foo")
-            ngx.say(val, " ", type(val))
-            val = dogs:get("bah")
-            ngx.say(val, " ", type(val))
-
-            local cats = ngx.shared.cats
-            val = cats:get("foo")
-            ngx.say(val or "nil")
-            val = cats:get("bah")
-            ngx.say(val or "nil")
-        ';
-    }
---- request
-GET /test
---- response_body
-32 number
-10502 number
-nil
-nil
---- no_error_log
-[error]
```

Comment:
I'm afraid you accidentally removed this test case in your merge, which is definitely undesired.